packages/ai/src/ONNXSegmentationController.ts

```diff
@@ -40,6 +40,8 @@ const ONNX_EVENTS = {
   MODEL_COMPONENT_LOADED: 'ONNX_MODEL_COMPONENT_LOADED',
 };
 
+export { ONNX_EVENTS };
```

Comment:
we don't export events like this in our packages, please check others like packages/core